src/Padawan/Domain/Completer/NamespaceCompleter.php

```diff
@@ -9,7 +9,7 @@
 class NamespaceCompleter extends AbstractFileInfoCompleter
 {
 
-    public function getEntries(Project $project, Context $context) {
+    public function getEntries(Project $project, Context $context, $cursorLine = 0) {
```

Comment:
agreed. don't know why I didn't come to this since it's a very natural design. Must be too much alcohol at new year's eve.